entity/response.py

```diff
@@ -1,10 +1,7 @@
-import json
-
-
 class Response(object):
     def __init__(self, body, status_code=200, err_message=None):
```

Comment:
Maybe change signature to `def __init__(self, status_code, body=None, err_message=None):` because you pass `status_code` explicitly everytime